WordPressKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "1.6.0-beta.1"
+  s.version       = "1.6.1-beta.1"
```

Comment:
If we're following semantic versioning, should we roll to `1.7.0-beta.1` instead of `1.6.1` as this is a new feature, not a bug fix?

Reply:
This was tricky for me - it's a Swift implementation of an existing endpoint, with a few more parameters passed. After a second look, I agree, I will update this to `1.7.0-beta.1`.